server/src/main/scala/controllers/application/exports/WithDesksExport.scala

```diff
@@ -34,18 +34,14 @@ trait WithDesksExport extends ExportToCsv {
     val start = SDate(startMillis.toLong)
     val end = SDate(endMillis.toLong)
     val summaryForPeriodFn = Exports.queueSummariesFromPortState(airportConfig.nonTransferQueues(terminal(terminalName)), 15, Terminal(terminalName), queryFromPortStateFn(None))
-    export(start, end, terminalName, summaryForPeriodFn)
+    Action(exportToCsv(start, end, "desks and queues", terminal(terminalName), Option((summaryActorProvider, GetSummaries)), summaryForPeriodFn))
```

Comment:
should we put dashes or underscores in the filename instead of spaces?

Reply:
It's actually used as both a file name and a description for logging, and has its spaces converted to dashes where it's used for the file name

Reply:
👍